themes/bootstrap3/less/vendor/autocomplete.less

```diff
@@ -1,10 +1,11 @@
 /* https://github.com/vufind-org/autocomplete.js (v2.1.3) */
 
-@autocomplete-item-bg   : #fff;
-@autocomplete-active-bg : @brand-primary;
-@autocomplete-hover-bg  : lighten(@brand-primary, 40%);
-@autocomplete-border    : lightgray;
-@autocomplete-secondary : darkgray; // item description
+@autocomplete-item-bg     : #fff;
```

Comment:
This file is an adaptation of the css provided in the repository. Since it only applies to the autocomplete, I thought this was a good place for it. If you'd rather not have it hear (since it's customized and would be edited outside of updates) we can put it in components.

Reply:
Yes, please move it to components. Putting something in a vendor folder should be taken to mean "this file came from somewhere else and thus cannot be edited." Since these files were locally created, I think components is a more appropriate home for them.